docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -206,16 +206,16 @@ You can define different variables for each individual host, or set shared varia
 
 .. _playbook_variables:
 
-Defining variables in a playbook
---------------------------------
+Defining variables in a play
+----------------------------
 
-You can define variables directly in a playbook::
+You can define variables directly in a playbook play::
```

Comment:
We could add that these variables will only be available for the current play only

Reply:
@bmillemathias tried to address that part with changes on L218. OK?

(Note, on review made a minor fix to wording on L218 and force updated this PR)

Reply:
yep, perfect